src/Sulu/Bundle/CategoryBundle/Resources/public/js/components/categories/list/main.js

```diff
@@ -37,7 +37,16 @@ define(function () {
                 toolbar: {
                     buttons: {
                         add: {},
-                        deleteSelected: {}
+                        deleteSelected: {},
+                        export: {
+                            options: {
+                                urlParameter: {
+                                    flat: true,
+                                    limit: 999999
```

Comment:
this have to be handled then in all controllers. i would use maybe a default value (9999999999999) in the csv-export method.